src/components/LibraryItem.tsx

```diff
@@ -20,26 +20,28 @@ import * as React from "react";
 import * as ReactDOM from "react-dom";
 import { ClusterView } from "./ClusterView";
 import { LibraryContainer } from "./LibraryContainer";
-import { ItemData, sortItemsByText } from "../LibraryUtilities";
+import * as LibraryUtilities from "../LibraryUtilities";
 
 export interface LibraryItemProps {
     libraryContainer: LibraryContainer,
-    data: ItemData,
+    data: LibraryUtilities.ItemData,
+    showExpandableToolTip: boolean,
     onItemWillExpand?: Function
 }
 
 export interface LibraryItemState {
-    expanded: boolean
+    itemExpanded: boolean,
+    toolTipExpanded: boolean
```

Comment:
If we end up renaming *item tooltip* to *item description*, then please restore `expanded` name.